great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -225,8 +225,15 @@ def _atomic_prescriptive_template(
             )
             header_params_with_json_schema.update(conditional_params)
 
-        quantiles = params["quantile_ranges"]["quantiles"]
-        value_ranges = params["quantile_ranges"]["value_ranges"]
+        quantile_ranges = params["quantile_ranges"] if params["quantile_ranges"] else {}
```

Comment:
Same question as before but if "quantile_ranges" is not in params, we'll have a KeyError here

Reply:
Good point. What about: 
```suggestion
        quantile_ranges = params.get("quantile_ranges") if params.get("quantile_ranges") else {}
```